protected/views/admin/generic/list.php

```diff
@@ -81,11 +81,16 @@
                 <?php
                 $retrieveResults = $admin->getSearch()->retrieveResults();
                 foreach ($retrieveResults as $i => $row) { ?>
+                    <?php if ($admin->getModelName()=="LetterType" && $admin->attributeValue($row, 'name')=="Internal Referral"):?>
+                        <tr class="" data-id="<?php echo $row->id ?>" onclick="new OpenEyes.UI.Dialog.Alert({content: 'Internal Referral type can not be changed.'  }).open();" >
```

Comment:
I'd change the wording to "The Internal Referral letter type cannot be changed.", but other than that it looks good

Reply:
@Zhaolian-Zhou If you do make the change Liam has described above, you should be able to rebase/merge it without needing us to review it once again.